### DISCUSSION TITLE
Welcome to DH-Full-Stack Discussions!

### DISCUSSION THREAD

**Discussion by mabregu** [mabregu/DH-Full-Stack#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asapconsulting** [comment]
Hello very good repository, what is the subject of this class?

**Reply by mabregu** [reply]
Hi how are you? If you refer to class 3, it is about Node.js modules, to export and import.

**Comment by pilot-solution** [comment]
Hello! Class 40 does not have express, why?

**Reply by mabregu** [reply]
It is because this class we focus on JS Front
